css/style.css

```diff
@@ -236,98 +236,201 @@ body {
     margin-left: -50px;
 }
 
+.profile{
+  background-color: var(--brickRed);
+}
+
 .profilePage_container {
-    background-color: var(--brickRed);
-    height: 100vh;
     color: white;
     font-family: "Fira Mono", monospace;
 }
 
-.profilePage_title p {
-    margin-bottom: 20px;
-    font-weight: bold;
-    font-family: "Fira Mono", monospace;
-    float: right;
-    margin-right: 290px;
-    position: relative;
-    bottom: 290px;
-}
-
 .profilePage_title h1 {
     font-family: "Roboto Slab", serif;
-    font-size: 304px;
-    opacity: 0.1;
+    font-size: 120px;
+    opacity: 0.3;
     text-align: center;
     position: relative;
-    top: -150px;
+    top: 10px;
     text-transform: uppercase;
 }
 
-.profilePage_userDetails h2 {
-    text-transform: uppercase;
-    font-family: "Fira Mono", monospace;
-    margin-left: 40px;
+/* for the bio */
+.profile-bio{
+  position: absolute;
+  right: 780px;
+  width: 300px;
+  height: 400px;
+  top: 50px;
+  /*border: 3px solid green;*/
+}
+
+.bio-card{
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  max-width: 300px;
+  margin: auto;
+  text-align: center;
+  background-color: #9c2626;
+}
+.bio-username {
+  color: grey;
+  font-size: 18px;
+}
+.bio-button {
+  border: none;
+  outline: 0;
+  display: inline-block;
+  padding: 8px;
+  color: white;
+  background-color: #6495ED;
+  text-align: center;
+  cursor: pointer;
+  width: 100%;
+  font-size: 18px;
+}
+.bio-button:hover {
+  opacity: 0.7;
+}
+
+/*for the content on the right */
+.main-content{
+  position: absolute;
+  right: 150px;
+  /*float: right;*/
+  width: 550px;
+  height: 400px;
+  top: 350px;
+  /*border: 3px solid green;*/
+}
+
+.profile-content {
+  position: absolute;
+  left: -70px;
+  /*right: 1px;*/
+  width: 700px;
+  height: 850px;
+  /*border: 3px solid yellow;*/
+}
+
+.profile-create{
+  position: absolute;
+  /*right: 150px;*/
+  width: 500px;
+  height: 1000px;
+  top: -20px;
+  /*border: 3px solid red;*/
+}
+
+.profile-tab {
+ position: absolute;
+ width: 600px;
+ height: 400px;
+ top: 430px;
+ /*border: 3px solid red;*/
+}
+#past-posts-id {
```

Comment:
why did you mix the styling by using both ids and class names? is it possible to only maintain class names as styling for the elements where possible? --- in this case, you wouldn't use the ids to style the element. use the class name.

Reply:
done